code/index.html

```diff
@@ -2,21 +2,201 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>THE GLOBETROTTER</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,300;0,500;1,500&family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+  
+    <!-- HEADER AND NAVBAR -->
+
+    <header> 
+      <img class="logo-navbar" src="img/logo.png" alt="globetrotter-logo">
+
+      <!-- For tablet only-->
+      <nav class="tablet">
+        <a href="#">TRAVEL NEWS</a>
+        <a href="#">DESTINATIONS</a>
+      </nav>
+      
+      <!-- For computer only-->
+      <nav>
+        <a href="#">WOMEN WHO TRAVEL</a>
+        <a href="#">TRAVEL NEWS</a>
+        <a href="#">THE BEST</a>
+        <a href="#">STORIES</a>
+        <a href="#">DESTINATIONS</a>
+      </nav>
     </header>
+    
+    <!-- NEWSLETTER CONTENT - only tablet and navbar -->
+
+    <section class="newsletter">
+      <h2 class="newsletter">Travel inspiration delivered right to your inbox</h2>
+      <button class="newsletter">SUBCRIBE NOW</button>
+      <button class="newsletter-tablet">SIGN UP FOR OUR NEWSLETTER</button>
+    </section> 
+
+
+    <!-- BIG NEWS CONTENT -->
 
     <section class="big-news">
-      <!-- The big news content -->
+      <img class="large" src="img/big-news.jpg" alt="countries-reopen">
+      <div class="big-news-text">
+        <a class="title-news" href="#">TRAVEL NEWS</a>  
+        <a class="intro-big" href="#">Coronavirus and travel: Everything you need to know</a>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- OTHER NEWS CONTENT  -->
+
+    <section class="other-news">
+
+      <div class="news-heading">
+        <h3 class="travel">Travel News</h3>  
+      </div>
+
+      <article class="news">
+
+        <div class="box-news">
+           <img class="small" src="img/dog-travel.jpg" alt="travel-dog">
```

Comment:
Consider using alt descriptions that are a bit more descriptive. An alternative text her could be "Dog standing on log bench". This text is what will be read out for people that are using screen readers and for some reason can't see the images 👓